gensim/utils.py

```diff
@@ -2025,3 +2025,20 @@ def lazy_flatten(nested_list):
                 yield sub
         else:
             yield el
+
+
+def save_as_line_sentence(corpus, filename):
+    """Save the corpus in LineSentence format, i.e. each sentence on a separate line,
+    tokens are separated by space.
+
+    Parameters
+    ----------
+    corpus : iterable of iterables of strings
+
+    """
+    with smart_open(filename, mode='w', encoding='utf8') as fout:
```

Comment:
should be `wb`

Reply:
also, this is a good idea to have a distinct test for it (for language like ru too).